x/signal/keeper_test.go

```diff
@@ -20,6 +24,52 @@ import (
 	tmdb "github.com/tendermint/tm-db"
 )
 
+func TestGetVotingPowerThreshold(t *testing.T) {
```

Comment:
Thanks for moving tests to `keeper_test.go`! I just find it easier to find the test if a function is defined in foo.go then the unit test is in foo_test.go.

I'll revert the change from https://github.com/celestiaorg/celestia-app/pull/3277 and/or rebase after this merges